Dockerfile

```diff
@@ -1,6 +1,9 @@
-FROM ghcr.io/navikt/pdfgen:2.0.28
+FROM ghcr.io/navikt/pdfgen:2.0.43
 
-ENV JDK_JAVA_OPTIONS="-XX:MaxRAMPercentage=75 -XX:InitialRAMPercentage=75"
+ENV JDK_JAVA_OPTIONS="-XX:MaxRAMPercentage=75 -XX:InitialRAMPercentage=75 -Dlogback.configurationFile=logback-remote.xml"
+# use the following two lines to enable testing in dev-gcp
```

Comment:
De to linjene under må kommenteres inn når man ønsker å teste via dev-ingressen fra utviklermaskin